hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/store/accessor/TokenDatabaseAccessor.java

```diff
@@ -148,6 +149,9 @@ private Supplier<Account> getAutoRenewAccount(Long autoRenewAccountId, final Opt
                 .map(t -> entityRepository.findActiveByIdAndTimestamp(autoRenewAccountId, t))
                 .orElseGet(() -> entityRepository.findByIdAndDeletedIsFalse(autoRenewAccountId))
                 .map(autoRenewAccount -> new Account(
+                        autoRenewAccount.getEvmAddress() != null
+                                ? ByteString.copyFrom(autoRenewAccount.getEvmAddress())
```

Comment:
Avoid extra allocations. Same comment for other change in file.

```suggestion
                                ? DomainUtils.fromBytes(autoRenewAccount.getEvmAddress())
```